src/components/DocumentControls/DocumentControls.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../constants/styles';
+
+.documentControlToggle{
+    padding: 5px 7px 0px;
+    height: 20px;
+
+    left: 50%;
+    width: 10px;
+    padding: 5px 15px 0px;
+    border-top-right-radius: 7px;
+    border-top-left-radius: 7px;
+    background-color: rgba(0, 0, 0, 0.2);
+
+    transform: translateY(-100%)translateX(-50%);
+    position: absolute;
```

Comment:
I think other sass files use 2 spaces so let's use 2 spaces here as well

Reply:
I updated it to be two spaces as well